change-beta/@azure-communication-react-d54219e1-a29e-44dc-a5ba-a9a4e4193856.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
one change file per PR is sufficient (unless there is a time when multiple things are introduced in the same PR, but then it really should be multiple PRs)